### PR TITLE
Fix for systray issue and adding msi build creation

### PR DESCRIPTION
**Pull request by Delaire**
for now parameters are passed using WRAPPED_ARGUMENTS from msiexe to exe, to see how this could be improved in a second version?

### CODE REVIEW

File: gui/SettingsForm.cs [hunk 1]
```diff
@@ -206,5 +229,34 @@ private void systray_MouseDoubleClick(object sender, MouseEventArgs e)
             Show();
             WindowState = FormWindowState.Normal;
         }
+
+        private void CheckRegistryForConfID()
+        {
```
Comment (rs) [inline gui/SettingsForm.cs:234]: The service should be the one checking for this and passing on the info to the UI. Is there a specific reason to have the clicking checking for the regkey by itself?

File: nsis/NextDNSSetup.nsi [hunk 2]
```diff
@@ -95,13 +95,51 @@ Section "NextDNS Service"
...
+  ; pop $R0
+  ; ${If} $R0 != ""
+    ; DetailPrint "Configuring profile $R0"
+    ; WriteRegStr HKLM "Software\NextDNS\Settings" "Profile" "$R0"
+  ; ${EndIf}
+
+  !insertmacro GetParameterValue "/CONF_ID=" ""
+  pop $R1
+  ${If} $R1 != ""
+	${If} ${RunningX64}
```
Comment (rs) [inline nsis/NextDNSSetup.nsi:108]: We may write any registry as the service is checking all of them: https://github.com/nextdns/windows/blob/095dd237e4061a1e5d7560bc63371d97457dd1aa/service/settings/registry.go#L47

File: gui/SettingsForm.cs (hunk 1 above)
Comment (Delaire) [reply gui/SettingsForm.cs:234]: My bad, I did not see that the go service was looking for a regkey and passing it, that is why I added it to the client UI, i'll update the PR and the nsi

**Comment by Delaire** [comment]
Will not merge

**Pull request closed without merging**